build/tools/SharedCodeGenerator/Outputs/CSharpOutput.cs

```diff
@@ -45,13 +48,21 @@ public string GetPath()
 
         public string GetContent()
         {
+            var scope = "public";
```

Comment:
Tried keeping default as internal but that is breaking existing code and will cause even more changes in this PR.

Reply:
so all constants are going to be public now .. if we need to change some to internal or protected we need to manually change them after the constants are generated?